spot_rl_experiments/spot_rl/real_policy.py

```diff
@@ -43,8 +44,21 @@ def __init__(
     ):
         print("Loading policy...")
         self.device = torch.device(device)
-        self.is_torchscript_policy = "torchscript" in checkpoint_path
+        config = construct_config()
+        self.is_torchscript_policy = "torchscript" in str(config.WEIGHTS_TYPE).lower()
+
         if self.is_torchscript_policy:
+            # map pytorch weight path to torchscript path based on basename
+            checkpoint_path_without_ext = os.path.basename(checkpoint_path).split(".")[
+                0
+            ]
+            checkpoint_path = [
+                torchscript_checkpoint_path
+                for torchscript_checkpoint_path in list(
+                    config.WEIGHTS_TORCHSCRIPT.values()
+                )
+                if checkpoint_path_without_ext in torchscript_checkpoint_path
+            ][0]
```

Comment:
This logic might still fail for mobile manipulation environment as `MIXER` weights are not under `TORCHSCRIPT`.

Reply:
Seems like MixerPolicy is already not working